.github/workflows/pylint.yml

```diff
@@ -0,0 +1,13 @@
+name: Python Code Quality and Lint
+on:
+  push:
+    branches: [ develop ]
+
+jobs:
+  pylint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: ricardochaves/python-lint@v1.3.0
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think we need a newline here :nerd_face:

Reply:
done! ... also I have seen this kind of practice before, but don't know the motive behind it. Can you point out why we need a newline? Thanks! :)